app/_hub/kong-inc/correlation-id/overview/_index.md

```diff
@@ -110,6 +110,10 @@ To edit your Nginx parameters, do the following:
 
    You should now see Correlation ID entries in the access log.
 
+
+{:.note}
+> **Note**: The `log_format` directive must be added inside the HTTP context of the Nginx configuration file. Failing to do so may cause Kong to fail on startup due to invalid Nginx configuration.
```

Comment:
This should provide a bit of info on what "HTTP content of the Nginx config file" means. 

You could go one of two ways here: either link to https://docs.konghq.com/gateway/3.8.x/production/logging/customize-gateway-logs/, which has a example at the bottom of the page that shows where log_format goes, or add code snippet that looks something like this:

```
http {
...
    log_format show_everything '$remote_addr - $remote_user [$time_local] '
        '$request_uri $status $body_bytes_sent '
        '"$http_referer" "$http_user_agent"';

}
```
Not sure if that option is desired, as the `...` seems to confuse people more often than not.

Reply:
Moved the note and updated the example with `http`.